### ISSUE TITLE
Recording. Recording Volume handle could be moved despite it being blocked by chosen audio host

### ISSUE THREAD

**Issue by Penikov** [audacity/audacity#2501]
**Describe the bug**
Minor, but it looks a little confusing for beginners. 

In case an appropriate audio host was chosen (i.e. MME) Recording  Volume handle will be unblocked and able to adjust recording level. 
But
If were chosen another audio host, which makes Recording Level inaccessible to a user, there is a workaround to make the handler move while recording is on:

**To Reproduce**

Chose Windows Direct Sound host

1. Run recording
2. Try to move the Playback Volume handle
3. Stop then run recording one more time
4. Try to pull the  Playback Volume handle again while recording is running
![image](https://user-images.githubusercontent.com/25851359/151951185-ccfe1da6-b317-4a15-982e-51fe07c3043f.png)

Please note: the Recording Volume handler _does not_ change volume levels as expected. This is just a little UI issue.

Another thing to be noted is the handle hasn't been set up to maximum automatically despite the Manual said: https://manual.audacityteam.org/man/mixer_toolbar.html#output

**Expected behavior**
It would be nice to have a friendly notification when a user tries to adjust the Recording Volume handle when the recording slider does not have direct control.

**Additional information (please complete the following information):**
 - OS:  Windows 10
 - Version: 3.1.3

**Comment by LWinterberg** [comment]
this won't be changed any time soon
